CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2022-08-24
+df0c367f021a6ced7b9039cbcb580f7ab33e9a42
+
+### Added
+- Users can collect works on liker.land, no need to redirect to app.like.co.
```

Comment:
Or maybe just say we improved the collect flow?

Reply:
Yes, I think simple is better

I think should use Changed instead of Added

Ref: https://keepachangelog.com/en/1.0.0/